lib/api/core/plugins/pluginsManager.js

```diff
@@ -61,6 +61,22 @@ class PluginsManager {
     this.strategies = {};
     this.routes = [];
 
+    // Structure:
+    // {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ACommentedCode)